hexkit/providers/akafka/testutils.py

```diff
@@ -364,6 +365,31 @@ def record_events(self, *, in_topic: Ascii) -> EventRecorder:
 
         return EventRecorder(kafka_servers=self.kafka_servers, topic=in_topic)
 
+    def delete_topics(self, topics: Optional[Union[str, list[str]]] = None):
+        """Delete given topic(s) from Kafka broker.
+
+        This process deletes the contained messages as the topics will be recreated
+        by the default config.
```

Comment:
When not spec
```suggestion
        When no topic(s) are specified, all existing topics will be deleted.
```

Reply:
Also part of [b7d4826](https://github.com/ghga-de/hexkit/pull/59/commits/b7d482637e1285499a7258a4b9f44ad3b2b63bcc)